src/mumble/CMakeLists.txt

```diff
@@ -269,6 +269,10 @@ set(MUMBLE_SOURCES
 	"XMLTools.cpp"
 	"XMLTools.h"
 
+	"dialogs/PositionalAudioViewer.cpp"
+	"dialogs/PositionalAudioViewer.h"
+	"dialogs/PositionalAudioViewer.ui"
```

Comment:
Not sure if it is justified to have a separate `dialogs` sub-directory just for this. Mainly because we have a bunch of other dialogs that I would logically also expect to be in that directory, should it exist.
I tend to keeping stuff in the main dir for now and then bring some order into the chaos at some point (but consistently for all files at once) :thinking:

Reply:
The reason I created `dialogs` is indeed to eventually put all related files inside.

However, if you prefer, I can move the new files outside and then we can move all of them in a separate PR.

Reply:
Yeah I think a separate clean-up would be best :eyes: